portfolio/src/main/webapp/script.js

```diff
@@ -185,15 +184,15 @@ function toggleHeatmap() {
 }
       
 function getPoints() {
-    return [
-          /*new google.maps.LatLng(reports[0][1], reports[0][2]),
-          new google.maps.LatLng(reports[1][1], reports[1][2]),
-          new google.maps.LatLng(reports[2][1], reports[2][2]),
-          new google.maps.LatLng(reports[3][1], reports[3][2]),
-          new google.maps.LatLng(reports[4][1], reports[4][2]),
-          new google.maps.LatLng(reports[5][1], reports[5][2])*/
-          
-    ];
+    var heatPoints = [];
+    var individualPoint;
+        fetch('/markers').then(response => response.json()).then((markers) => {
```

Comment:
the tabbing is incorrect and makes it very difficult to read. It should be
```suggestion
    fetch('/markers')
        .then(response => response.json())
        .then((markers) => {
```